ui/main/src/app/modules/logging/components/logging-filters/logging-filters.component.html

```diff
@@ -35,14 +35,9 @@
                                         labelKey="archive.filters."></of-datetime-filter>
                 </div>
             </div>
-            <div  style="margin-top:125px; margin-left:60px;min-width: 320px; ">
-                <div class="form-group">
-                    <div class="row">
-                        <button id="opfab-logging-btn-search" class="opfab-btn" (click)='sendQuery()' translate> button.search</button>
-                        <button id="opfab-logging-btn-cancel" class="opfab-btn-cancel" (click)='resetForm()' translate> button.reset</button>
-                      
-                    </div>
-                </div>
+            <div  class="opfab-buttons">
+                <button id="opfab-logging-btn-search" class="opfab-btn" (click)='sendQuery()' translate> button.search</button>
+                <button id="opfab-logging-btn-cancel" class="opfab-btn-cancel" (click)='resetForm()' translate> button.reset</button>  
```

Comment:
Set  "style="margin-top: 50px;"  to cancel button  to align search button with form fields, like in archives screen

Reply:
Done